ops/infrastructure/bastion_playbook.yml

```diff
@@ -345,77 +187,37 @@
 
   - name: Create a backups directory for storing database dump files
     ansible.builtin.file:
-      path: /home/centos/files-url-updater/backups
+      path: "{{ item }}"
       state: directory
       mode: '0755'
       owner: centos
       group: centos
-
-  - name: Install create and upload database backup to S3 script for file url updater
-    ansible.builtin.copy:
-      src: ../../../../gigadb/app/tools/files-url-updater/upload_database_backup_to_S3.sh
-      dest: /home/centos/files-url-updater/
-      owner: centos
-      group: centos
-      mode: a+x
+    loop:
+      - "/home/centos/backups"
 
   - name: Setup cronjob to automate copying database backup file to S3
     ansible.builtin.cron:
       name: "Create database dump file from RDS and upload to S3 every day"
       minute: "00"
       hour: "11"
       user: "centos"
-      job: "cd /home/centos/files-url-updater ; ./upload_database_backup_to_S3.sh 2> $HOME/logs/upload_dump_s3_errors.log 1> $HOME/logs/upload_dump_s3_output.log"
+      job: "docker run --rm --env-file .env -v /home/centos/backups:/backups -v /home/centos/.config/rclone/rclone.conf:/root/.config/rclone/rclone.conf registry.gitlab.com/{{ gitlab_project }}/production_s3backup:{{ gigadb_environment }}"
```

Comment:
Hi @rija,

Would it be useful to have `$HOME/logs/upload_errors.log` and `$HOME/logs/upload_status.log` for this step?

Reply:
Hi @kencho51,
since the crontab entry is just a docker command, we can use docker logging to get the log from the process.
However the use of ``--rm`` deletes the container immediately after its run, thus preventing access to the log.
I've now corrected that in my last commit. Now we can access the command's log:
```
[centos@ip-10-99-0-172 ~]$ crontab -l
#Ansible: Reset database every day after a new backup is available on the ftp server
05 10 * * * ./databaseReset.sh 2> $HOME/logs/errors.log 1> $HOME/logs/output.log
#Ansible: Create database dump file from RDS and upload to S3 every day
00 11 * * * docker run --env-file .env -v /home/centos/backups:/backups -v /home/centos/.config/rclone/rclone.conf:/root/.config/rclone/rclone.conf registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging
[centos@ip-10-99-0-172 ~]$ date
Fri Aug 26 16:57:35 UTC 2022
[centos@ip-10-99-0-172 ~]$ crontab -e
crontab: installing new crontab
[centos@ip-10-99-0-172 ~]$ date
Fri Aug 26 16:57:52 UTC 2022
[centos@ip-10-99-0-172 ~]$ docker system df
TYPE            TOTAL     ACTIVE    SIZE      RECLAIMABLE
Images          10        1         3.129GB   3.081GB (98%)
Containers      4         0         0B        0B
Local Volumes   1         0         494.5MB   494.5MB (100%)
Build Cache     0         0         0B        0B
[centos@ip-10-99-0-172 ~]$ date
Fri Aug 26 17:01:12 UTC 2022
[centos@ip-10-99-0-172 ~]$ docker ps -a
CONTAINER ID   IMAGE                                                                                   COMMAND                  CREATED          STATUS                      PORTS     NAMES
552918362e9d   registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging   "/upload_database_ba…"   3 minutes ago    Exited (0) 2 minutes ago              fervent_noether
8d617200bceb   registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging   "bash -c 'echo hello…"   14 minutes ago   Exited (0) 14 minutes ago             zealous_colden
3550b5587818   registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging   "bash -c 'echo hello…"   15 minutes ago   Exited (0) 15 minutes ago             nice_bose
29bb0837773c   registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging   "bash -c 'echo fooba…"   17 minutes ago   Exited (0) 17 minutes ago             nice_brown
cffe67eab4c0   registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_s3backup:staging   "bash -c 'echo boo'"     19 minutes ago   Exited (0) 19 minutes ago             s3backup
[centos@ip-10-99-0-172 ~]$ docker logs 552918362e9d
/usr/bin/rclone
 pg_terminate_backend 
----------------------
(0 rows)

```

Reply:
👍🏻